tools/ci_build/requirements/requirements-pybind.txt

```diff
@@ -0,0 +1,8 @@
+setuptools
```

Comment:
Should be tools/ci_build/requirements/pybind/requirements.txt . The filename must be requirements.txt.